### DISCUSSION TITLE
Updating Codespace with push from loal

### DISCUSSION THREAD

**Discussion by ernest-phillips** [community/community#26923] [Codespaces]
<p>I already spun up a Codespace on a repo, but I made some changes locally on my machine.<br>
Do you need to delete the codespace and rebuild it to incorporate those changes?</p>

**Comment by Chuxel** [comment, marked as answer]
<p>If you push them into the repository (or in a fork or branch), you can open the Codespace and use either it’s built-in source control support or git from the command line or to pull in the changes. No need to recreate!</p>

**Comment by ernest-phillips** [comment]
<p>Fantastic! Not sure why that didn’t cross my mind. Loving the feature otherwise.</p>

**Comment by Chuxel** [comment]
<p>Great! As a FYI, you can also drag-and-drop any files from your desktop into the explorer to copy them to the Codespace (even on the web). There’s also a right click gesture to download a folder from a Codespace.  I mention git since it can be easier to do if you’ve got changes in both places.</p>
